diff -up kfind-20.03.90/src/kftabdlg.cpp.omv~ kfind-20.03.90/src/kftabdlg.cpp
--- kfind-20.03.90/src/kftabdlg.cpp.omv~	2020-04-13 01:47:57.061039199 +0200
+++ kfind-20.03.90/src/kftabdlg.cpp	2020-04-13 01:48:03.212107949 +0200
@@ -286,11 +286,7 @@ KfindTabWidget::KfindTabWidget(QWidget *
     connect(bg, static_cast<void (QButtonGroup::*)(QAbstractButton *)>(&QButtonGroup::buttonClicked), this, &KfindTabWidget::fixLayout);
     connect(sizeBox, static_cast<void (KComboBox::*)(int)>(&KComboBox::activated), this, &KfindTabWidget::slotSizeBoxChanged);
     connect(timeBox, static_cast<void (QSpinBox::*)(int)>(&QSpinBox::valueChanged), this, &KfindTabWidget::slotUpdateDateLabelsForNumber);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(betweenType, static_cast<void (KComboBox::*)(int)>(&KComboBox::currentIndexChanged), this, &KfindTabWidget::slotUpdateDateLabelsForType);
-#else
-    connect(betweenType, static_cast<void (KComboBox::*)(int, const QString &)>(&KComboBox::currentIndexChanged), this, &KfindTabWidget::slotUpdateDateLabelsForType);
-#endif
     connect(sizeEdit, static_cast<void (QSpinBox::*)(int)>(&QSpinBox::valueChanged), this, &KfindTabWidget::slotUpdateByteComboBox);
 
     // ************ Page Three
